web_src/js/components/ContextPopup.vue

```diff
@@ -102,16 +95,8 @@ export default {
       <p><small>{{ issue.repository.full_name }} on {{ createdAt }}</small></p>
       <p><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</p>
       <p>{{ body }}</p>
-      <div class="labels-list">
-        <div
-          v-for="label in labels"
-          :key="label.name"
-          class="ui label"
-          :style="{ color: label.textColor, backgroundColor: label.color }"
-        >
-          {{ label.name }}
-        </div>
-      </div>
+      <!-- eslint-disable-next-line vue/no-v-html -->
+      <div v-html="renderedLabels"/>
```

Comment:
Is that safe?
After all, it contains user-supplied content…

Reply:
I assume the backend sanitizes this during `templates.RenderLabels` but good to double check.

Reply:
It is safe, because it is provided by `templates.RenderLabels`. `templates.RenderLabels` is also directly used in other templates, so if there is anything wrong, it should have already been wrong for long time.

ps: I also reviewed the `RenderLabels / RenderLabel` related code in their PRs, and at that time I already pointed out some XSS problems and they were fixed.

Reply:
Although the old code is safe enough, we could make it easier to read & maintain by using `HTMLFormat`: bf338dbb6f2062356f9dcff43b2a16640379e0ff

Reply:
![image](https://github.com/go-gitea/gitea/assets/2114189/a79597b6-7992-4046-95ca-3189ac7fa1ff)

Reply:
Oops, it's merged by auto-merge. Sorry for that the new commit hasn't been reviewed. Feel free to do more reviews, and I will fix the problems in first time. 🙏